CondFormats/PCLConfig/plugins/AlignPCLThresholdsWriter.cc

```diff
@@ -78,7 +74,9 @@ AlignPCLThresholdsWriter::~AlignPCLThresholdsWriter() { delete myThresholds; }
 void AlignPCLThresholdsWriter::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup) {
   using namespace edm;
 
-  edm::LogInfo("AlignPCLThresholdsWriter") << "Size of AlignPCLThresholds object " << myThresholds->size() << std::endl
+  AlignPCLThresholds myThresholds{};
+
+  edm::LogInfo("AlignPCLThresholdsWriter") << "Size of AlignPCLThresholds object " << myThresholds.size() << std::endl
```

Comment:
just noticing there are now two `std::endl` (I can take care of it in the next update).

Reply:
I would have this PR merged now as part of the PoolDBOutput migration.
And then if you can take care of this cleanup later it would be great, thanks @mmusich !